src/components/ha-sidebar.html

```diff
@@ -197,6 +201,10 @@
     this._boundUpdateStyles = this.updateStyles.bind(this);
   },
 
+  tr: function (pg, text, options) {
```

Comment:
Cool. The Polymer 2 changes will be a big help here then.

Also yeah, I agree about the libs. It's definitely larger, but it also covers more features like dates and numbers. I think we've done pretty well using the native locale functions, but anywhere we'd have numbers/dates embedded in a string we'd otherwise hit a roadblock.

I think for me since they both have reasonable pros and cons, the tipping point is let's just stick with the Polymer recommended functions.